views/building-hours/all-building-hours.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {
+  Text,
+} from 'react-native'
+
+import moment from 'moment-timezone'
+const CENTRAL_TZ = 'America/Winnipeg'
+
+export function allBuildingHours(info, style) {
+  let dayTimes = []
+  let current = moment().tz(CENTRAL_TZ)
+
+  for (let i = 0; i < 7; i++) {
+    let hoursString = ''
+    let day = current.format('dddd')
+    let d = current.format('ddd')
+
+    current = current.add(1, 'days')
```

Comment:
Why are we adding a day to `current` here when it doesn't appear to be used again?